### PR TITLE
Use editable area for text and links

### PR DESCRIPTION
**Pull request by motechFR**
Use the existing component, and autoexpand values

### CODE REVIEW

**Comment by mattcasey** [comment]
Please fix the text so that it's the smaller font size we had prior:
Current:
image
New one (too big):
image

File: components/common/BoardEditor/focalboard/src/components/properties/link/link.tsx [hunk 1]
```diff
@@ -34,11 +34,10 @@ function URLProperty(props: Props): JSX.Element {
   return (
     <div className='URLProperty property-link url'>
       {(hasValue || props.placeholder) && (
-        <Editable
```
Comment (mattcasey) [inline components/common/BoardEditor/focalboard/src/components/properties/link/link.tsx:37]: maybe you just need to brign back this class `octo-propertyvalue`

Comment (motechFR) [reply components/common/BoardEditor/focalboard/src/components/properties/link/link.tsx:37]: octo-propertyvalue definitely cleaned up a bunch of the text

**Comment by mattcasey** [comment]
the labels and input are out of line still:
image

**Comment by motechFR** [comment]
@mattcasey fixed this, so we do a center align in case of empty content. Let me know if that works

**Pull request closed without merging**